File: src/model/Notification.java
package model;

public class Notification {

	private String message;
	private boolean isModal;
	
	public Notification(String message) {
		this(message, false);
	}
	
	public Notification(String message, boolean isModal) {
		this.message = message;
		this.isModal = isModal;
	}
	
	public String getMessage() {
		return message;
	}
	
	public boolean getIsModal() {
		return isModal;
	}
}

File: web/src/model/Notification.ts
export class Notification {
  private message: string;
  private isModal: boolean;

  constructor(message: string, isModal: boolean = false) {
    this.message = message;
    this.isModal = isModal;
  }

  getMessage(): string {
    return this.message;
  }

  getIsModal(): boolean {
    return this.isModal;
  }
}
